### ISSUE TITLE
Automatic test that parameters to `LOGGER.*` are not localized

### ISSUE THREAD

**Issue by koppor** [koppor/jabref#137]
Maybe similar to net.sf.jabref.ArchitectureTests, but just checking that in the same line of `LOGGER.*` `Localization.lang` doesn't appear.

**Comment by oscargus** [comment]
This is related to something I have thought about for a while: how should we output stuff in the CLI? Now, I've kept `System.err/out` but most introduced stuff I've used `LOGGER`. I guess this issue means that everything in the CLI should use `System.err/out` as one would like translated output there?

**Comment by koppor** [comment]
Yes, CLI via System.out. LOGGER is IMHO more for debugging purposes.
